release-notes/Cloud/DCP_change_log_2025.md

```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 6 March 2025 - Fix - Catalog - Deploy button state when not authenticated [ID 42439]
+
+When users are not authenticed, the state of the deploy button will now be applied correctly if their issues with the version you want to deploy.
```

Comment:
For a fix, the description should clearly describe what the issue is, rather than what the situation now is. Users looking in the fixes will be searching for what they encountered, so we need to make sure this can easily be spotted. Could you rephrase both this item and the item below so it's a clearly recognizable description of the issue?